### ISSUE TITLE
500 error when searching AAPB website

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1275]
Please make this your highest priority, not just fixing it but investigating why this keeps happening.

FYI - if this has anything to do with increased traffic, we had only 566 sessions on the site on Tuesday (June 13, 2017), and 588 on Wednesday (June 14, 2017). This is about 1.5 to 2 times higher than our daily average traffic.

**Comment by jasoncorum** [comment]
I spent today solving the logrotate issue we’ve been having on AAPB. I believe I found the issue and tested a fix on the demo site to make sure the fix actually worked. However, I didn’t want to unilaterally put in the fix because sysadmin stuff isn’t a huge strength of mine and I want your opinions on best way to accomplish solving the issue. Unfortunately, I’m out on vacation starting tomorrow, so here are my findings and hopefully you all can put the final fix in place w/out me.
 
So, I think the key issue and solution is outlined in this StackOverflow thread:
https://stackoverflow.com/questions/6540279/logrotate-does-not-auto-rotate-when-based-on-log-size
 
Essentially, even though the script specifies to rotate once the log reaches a certain size it still won’t run until logrotate runs, which if I remember from my reading today, only runs once a day.
 
Since this isn’t exactly our desired behavior, here’s what I successfully set up on demo and then commented out once I verified it worked. I essentially edited the root user’s crontab so that Kevin’s script ran once a minute. I had to do this as the root user since our regular user account doesn’t have write access to: var/log/httpd.  I think we could set this to run once an hour and we’d be good.
 
What smells a little to me is having to use the root account’s crontab, but I’m not sure if there’s a good workaround for it. Thoughts?
